server/remote_cache/action_cache_server/action_cache_server.go

```diff
@@ -153,10 +154,16 @@ func (s *ActionCacheServer) GetActionResult(ctx context.Context, req *repb.GetAc
 	downloadTracker := ht.TrackDownload(d)
 	blob, err := s.cache.Get(ctx, rn.ToProto())
 	if err != nil {
-		ht.TrackMiss(d)
+		if htErr := ht.TrackMiss(d); htErr != nil {
```

Comment:
This LGTM but the prefixed errors are unconventional/weird :P `if err := ...; err != nil { ... }` is more idiomatic. If you're worried about clobbering other `err` variables in the same scope, the `err` is only scoped to the `if` block so that's not an issue.

Reply:
The initial implementation is pretty much just me spelunking for Redis-related issues, so it was named that way to avoid clobbering.

Now that we added the `if` clause, it made more sense to use `err` like you suggested.

Pushed a change for this